contrib/kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml

```diff
@@ -74,6 +79,38 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+
+        {{- if .Values.exporters.jmx.enabled }}
+        - name: jmx-exporter
```

Comment:
Is it possible to extract this out into a library chart? https://helm.sh/docs/topics/library_charts/ Seems like a lot of the code needs to be copy and pasted

Reply:
Sure! I'll check that.

Reply:
Thanks for the contribution @bernardino, please let us know once you've been able to refactor this into a library chart. Looks great otherwise.

Reply:
It's running the build at the moment.

@dexter-mh-lee it looks waaay cleaner now! Thanks for your suggestion!

And I'm sorry for the delay.